README.md

```diff
@@ -23,6 +23,17 @@
   [![Sponsors on Open Collective](https://opencollective.com/nest/sponsors/badge.svg)](https://opencollective.com/nest#sponsor)-->
 
 ## Description
+This E-commerce API is designed for stores to have online presence and for online shopping. The platform allows users to browse products, add items to their cart, and proceed to checkout. It includes user authentication, role-based access control (e.g., RETAIL_ADMIN,SUPER_ADMIN and USER).
```

Comment:
You can create a place to show routes and stuff like that. I think that'll help people know what routes to call. 

Also we can show why nest was picked as the technology to work with showing in details what the project is about and how nest can play a crutial role in the development of such platform

Reply:
okay, noted!